data/speakers.ts

```diff
@@ -327,14 +327,13 @@ const speakers: Speaker[] = [
   {
     objectID: 'siddharthkp',
     fullName: 'Siddharth Kshetrapal',
-    tags: ['CSS', 'Web Performance', 'React', 'CSS-in-JS', 'Node', 'Testing'],
+    tags: ['React', 'Design Systems', 'Web Performance'],
     email: null,
     website: 'https://sid.studio/',
     social: {
       twitter: 'siddharthkp',
       github: 'siddharthkp',
-      blog: 'https://medium.com/@siddharthkp',
-      linkedin: 'https://www.linkedin.com/in/siddharthkp/',
+      blog: 'https://sid.studio/newsletter'
     },
     talks: [],
```

Comment:
Hi @siddharthkp, thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Hi, I'm merging this for now, but could you please update your talks in the new PR? Thank you.